docs/experiments-plus/read-results.md

```diff
@@ -13,19 +13,16 @@ You can also deep dive into how all your metrics shifted with the experiment.
 #### Key Results
 The **Key Results** panel shows how your primary metrics compare against the control for each variant.
 
-In the example below, the **Circle** variant is performing better than the **Rounded Square (control)** for the purchase count metric though this lift isn't statistically significant. 
-The **Square** variant is performing worse than the **Rounded Square (control)** and this difference is statistically significant. 
+In the example below, the **Square** variant is performing better than the **Rounded Square (control)** and **Circle** variant on all Key Metrics. 
    
-![image](https://user-images.githubusercontent.com/1315028/146469024-c1ac0344-4fe7-4596-b73e-212d045f8c63.png)
+![image](https://user-images.githubusercontent.com/1315028/148863280-3e7baf63-e184-466b-a0c2-8828e0f706df.png)
 
 
 #### Metrics Lifts
-The **Metrics Lifts** panel shows how your favorited metrics compare against the control for each variant.  
-   
-![image](https://user-images.githubusercontent.com/1315028/146469433-5e6ef872-ef5f-478a-8482-b9ec9452c28a.png)
-
+The **Metrics Lifts** panel shows how your *favorited* metrics compare against the control. You can also see how *all* your other metrics compare against the control in the **All Other Metrics** below the **Favorited Metrics** section as shown below.
```

Comment:
compared*

the **All Other Metrics** (section) below

End with "as shown here" to remove duplicate "below"